test/statmodels.jl

```diff
@@ -82,6 +82,12 @@ ct = CoefTable(m, ["Estimate", "Stderror", "df", "p"], [], 4)
      df = 0.2422083248151139, p = 0.4530583319523316)
 ]
 
+ct = CoefTable(rand(15,100), ["c$i" for i in 1:100], [], 4)
+sct = sprint(show, ct, context=:limit=>false)
+@test textwidth(first(split(sct, '\n'))) > 900
+sct = sprint(show, ct, context=:limit=>true)
+@test textwidth(first(split(sct, '\n'))) <= displaysize()[2]
```

Comment:
Haven't you found a way to test the exact output as discussed above?

Reply:
The `displaysize()` returns current terminal dimensions, what ever it maybe in CI environment. I would assume standard 80x25.

Reply:
Yes I think it's same to assume that, and check the string output for these dimensions.